torch/distributed/fsdp/_shard_utils.py

```diff
@@ -13,6 +13,8 @@
     TensorProperties,
 )
 from torch.distributed._shard.sharding_spec import ShardMetadata
+from torch.distributed._tensor import DTensor, Shard as DShard
```

Comment:
why we need to rename this as `DShard`? the name is kinda weird, is it because it's conflicting with ShardedTensor's Shard?

Reply:
Ye. In the same file, it also uses Shard for ShardedTensor. We can rename it from DShard to Shard when we kill the sharded_tensor logics.